modules/kits-elements-defaults/assets/js/editor/commands/create.js

```diff
@@ -0,0 +1,40 @@
+import localSettingsExtractor from '../settings-handlers/local/extract';
+
+export default class Create extends $e.modules.CommandBase {
+	settingsExtractors = [
+		localSettingsExtractor,
+	];
+
+	validateArgs( args = {} ) {
+		this.requireContainer();
+	}
+
+	async apply( { container } ) {
+		$e.internal( 'panel/state-loading' );
+
+		const type = container.model.get( 'widgetType' ) || container.model.get( 'elType' );
+
+		const settings = this.settingsExtractors.reduce( ( carry, extractor ) => {
+			return {
+				...carry,
+				...extractor( container ),
+			};
+		}, {} );
+
+		try {
+			await $e.data.create( 'kits-elements-defaults/index', { settings }, { type } );
+
+			// Refresh Cache.
+			// Cannot use {refresh: true} in `get()` because in the hooks there must be a way to get the data
+			// in sychronous way, and when using `refresh: true`, the data will not be available in syncronous way.
+			$e.data.cache.storage.removeItem( 'kits-elements-defaults' );
+			await $e.data.get( 'kits-elements-defaults/index' );
```

Comment:
Extract and use `loadDefaultValues` from the module.

Reply:
Cannot extract from module + feels like creating method to wrap a command is too much wrapping

Reply:
You already have a `loadDefaultValues()` function on:
https://github.com/elementor/elementor/pull/19778/files#diff-4715101938179d6613bb7a4945c287b4b4715602452c587dddeb3120e92c51afR9-R10

Reply:
Done